schemas/com.roku/video/jsonschema/1-0-0

```diff
@@ -0,0 +1,115 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Context Schema for a Roku video player event",
```

Comment:
I wonder if linking to some Roku documentation in the description might be useful to clarify some of the fields in here?

Reply:
That's a good idea! I added a link to the Video node documentation that this is based on.